The localization files are no longer compatible with
new gettext, and we are not installing them anyway.

--- a/Makefile.am
+++ b/Makefile.am
@@ -17,7 +17,6 @@
 SUBDIRS = \
 	include \
 	man-po \
-	po \
 	testsuite
 
 AM_CFLAGS = -Iproc
